src/lib/viewers/doc/PresentationViewer.js

```diff
@@ -228,10 +230,10 @@ class PresentationViewer extends DocBaseViewer {
             return;
         }
 
-        event.preventDefault();
-
         if (event.type === 'touchstart') {
             this.scrollStart = event.changedTouches[0].clientY;
+        } else if (event.type === 'touchmove') {
+            event.preventDefault();
```

Comment:
@pramodsum you don't use touchmove for anything correct? And @MinhHNguyen you prevent default behavior anyways?